python/generate/templates/api_client.mustache

```diff
@@ -0,0 +1,724 @@
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+import atexit
+import datetime
+from dateutil.parser import parse
+import mimetypes
```

Comment:
Add the `json` import back here. Do you think we should use `simplejson` the same as we do in the rest.py file or the standard json library? I vote to stick with the standard if we don't need `simplejson` here and we can evaluate its effectiveness for deserialisation separately.